<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertEmployee</class>
 <widget class="QDialog" name="InsertEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工注册</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>14</y>
     <width>335</width>
     <height>55</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>335</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;新员工注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Bonus">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>奖金：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="employee_Bonus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入奖金</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_departmentNum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>部门编号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="departmet_Num">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="department_Name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>研发部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>财务部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>行政部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>销售部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公关部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>仓管部</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_departmentName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>部门名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Position">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>职位：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="employee_Position">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>职员</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副部长</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>部长</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>总监</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>总经理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>董事长</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>总裁</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>112</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>152</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Again">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>115</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请确认密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_TEL">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电话号码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="employee_Tel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入电话号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sex">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>190</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Employee_Sex_Man">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Employee_Sex_woman">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Nationality">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>民族：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Employee_Nationality">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>汉族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>满族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>蒙古族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>基诺族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>瑶族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>土家族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>白族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>回族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>藏族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>维吾尔族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>苗族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>彝族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>壮族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>布依族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>侗族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哈尼族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>哈萨克族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>傣族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黎族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>傈傈族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>佤族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>畲族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高山族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>拉祜族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>水族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>东乡族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>纳西族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>景颇族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>柯尔克孜族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>土族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>达翰尔族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>仫佬族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>羌族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>布朗族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>撒拉族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>毛南族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>仡佬族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>锡伯族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>阿昌族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>普米族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>朝鲜族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>塔吉克族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>怒族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>乌孜别克族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>俄罗斯族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>鄂温克族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>德昂族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保安族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>裕固族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>京族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>塔塔尔族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>独龙族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>鄂伦春族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>赫哲族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>门巴族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>珞巴族</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>基诺族汉族</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="employee_Education">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>264</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入学历</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Education">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>学历：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Salary">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>36</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>工资：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="employee_Salary">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入工资</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Employee_Name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入姓名</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
